policyengine_us/parameters/gov/states/mi/tax/income/credits/home_heating_credit/standard_allowance/heating_cost_rate.yaml

```diff
@@ -0,0 +1,18 @@
+description: Michigan multiplies the home heating credit amount by this rate to compute the reduced standard credit.
+values:
+  2021-01-01: 0.5
+metadata:
+  period: year
+  unit: /1
+  label: Michigan home heating credit reduced standard allowance rate
```

Comment:
```suggestion
  label: Michigan home heating credit standard allowance rate if heating costs included in rent
```

Reply:
should it be standard allowance or standard credit? Either way we should consolidate everywhere